vip-cache-manager/ttl-manager.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+Plugin name: TTL Manager
+Description: Sets sane and reasonable cache TTLs for site responses.
```

Comment:
Perhaps more neutrally:

```
Description: Sets cache TTLs for site responses, see https://vip.wordpress.com/documentation/vip-go/controlling-vip-go-page-cache/
```

Incidentally we need to document this change in the client facing docs.